app/helpers/application_helper/toolbar/miq_ae_tools_simulate_center.rb

```diff
@@ -6,6 +6,7 @@ class ApplicationHelper::Toolbar::MiqAeToolsSimulateCenter < ApplicationHelper::
       N_('Copy object details for use in a Button'),
       N_('Copy'),
       :url       => "resolve",
-      :url_parms => "?button=copy"),
+      :url_parms => "?button=copy",
+      :klass     => ApplicationHelper::Button::ButtonWithoutRbacCheck),
```

Comment:
I can't find a purpose of this button, neither in documentation, @gmcculloug any idea what does the **Copy** button do?

<img width="791" alt="ss12" src="https://cloud.githubusercontent.com/assets/1187051/23127106/0e8c282e-f77a-11e6-95e4-56d16524f5b9.png">